### PR TITLE
Spark cde rename

### PR DESCRIPTION
**Pull request by tovganesh**
- adapter rename
- add new CDE API connection method
- update README to reflect CDE API profile

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,15 +29,18 @@ demo_project:
...
+     host: https://spark-cde-gateway.my.org.com/dex/api/v1/
      user: my_user
      password: my_pass
 ```
 
+- To obtain auth_endpoint follow the steps here: (https://docs.cloudera.com/data-engineering/cloud/api-access/topics/cde-api-get-access-token.html)
+
 ### Caveats
-- While using livy , in the Livy UI if you notice sessions change state to dead from starting instead of idle, make sure there is a proper mapping for the user in the IDBroker mapping section 
+- While using cde , in the Livy UI if you notice sessions change state to dead from starting instead of idle, make sure there is a proper mapping for the user in the IDBroker mapping section
```
Comment (TapasSenapati) [inline README.md:43]: Is this still relevant for cde

File: dbt/adapters/spark_cde/cdeapisession.py [hunk 2]
```diff
@@ -0,0 +1,457 @@
...
+    def createResource(self, resource_name, resource_type):
+        params = {'hidden': False, 'name': resource_name, 'type': resource_type}
+        # print('createResource - params', params)
+        res = requests.post(self.base_api_url + "resources", data=json.dumps(params), headers=self.api_header)
+        # print("createResource", res.text)
+        return res
+
+    def deleteResource(self, resource_name):
+        res = requests.delete(self.base_api_url + "resources" + "/" + resource_name, headers=self.api_header)
+        # print("deleteResource", res.text)
```
Comment (TapasSenapati) [inline dbt/adapters/spark_cde/cdeapisession.py:210]: Can we modify all these print statements to logs.info

File: dbt/adapters/spark_cde/connections.py [hunk 3]
```diff
@@ -84,6 +99,7 @@ class SparkCredentials(Credentials):
     server_side_parameters: Dict[str, Any] = field(default_factory=dict)
     retry_all: bool = False
     password: Optional[str] = None
+    auth_endpoint: Optional[str] = None
```
Comment (TapasSenapati) [inline dbt/adapters/spark_cde/connections.py:102]: Is this optional

Comment (tovganesh) [reply dbt/adapters/spark_cde/connections.py:102]: Not for CDE, but this also has livy code

File: README.md (hunk 1 above)
Comment (tovganesh) [reply README.md:43]: We need to see this. I am unsure at the moment as we have tested this with user with IDBroker mapping.

**Comment by tovganesh** [comment]
discarding this PR

**Pull request closed without merging**